include/seqan3/contrib/charconv/charconv_pre.hpp

```diff
@@ -0,0 +1,57 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include <cassert>
+#include <system_error>
+#include <limits>
+
+#include <seqan3/core/platform.hpp>
+
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_BEGIN_NAMESPACE_STD namespace seqan3::contrib::charconv {
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_END_NAMESPACE_STD }
+// #define SEQAN3_CONTRIB_CHARCONV_LIBCPP_HAS_NO_PRAGMA_SYSTEM_HEADER 1 // needs to be set if compiling with MSVC
```

Comment:
Why don't you wrap this in 
```
#if defined (_MSC_VER)
// ...
#endif
```
? (not sure if this macro is the right one)

Reply:
We don't support MSVC right now, so I think I don't need to do this. Fun fact: MSVC has full charconv support, so this header should not be included.